### PR TITLE
➕ Add GET request for leagues and teams

### PR DESCRIPTION
**Pull request by M-Sieger**
-  add samples.http
- add team.ts and league.ts

### CODE REVIEW

**Review by sweisz — APPROVED** [review]
LGTM 👍

File: pages/api/leagues.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (mhstrkmp) [inline pages/api/leagues.ts:1]:
```suggestion
```

File: pages/api/teams.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```
Comment (mhstrkmp) [inline pages/api/teams.ts:1]:
```suggestion
```

**Comment by HorstFrank** [comment]
Good Job!!

**Review by mrana17 — APPROVED** [review]
Good job 👏

**Pull request merged**